### PR TITLE
Calculate criminal tendency

### PR DESCRIPTION
**Pull request by Kubasinska**
Added criminal tendency calculator and other minor changes

### CODE REVIEW

File: mesa/Person.py [hunk 1]
```diff
@@ -266,8 +267,43 @@ def find_job(self):
...
+            self.family_link_neighbors())) > 0.5 else 1.0
+        # crim-neigh
+        self.criminal_tendency *= 1.81 if self.get_neighbor_list("friendship") or self.get_neighbor_list("professional") and (
+                len([agent for agent in self.get_neighbor_list("friendship") if agent.num_crimes_committed > 0]
+                    + [agent for agent in self.get_neighbor_list("professional") if agent.num_crimes_committed > 0]) / len(
+            [agent for agent in self.get_neighbor_list("friendship")] + [agent for agent in self.get_neighbor_list(
+                "professional")])) > 0.5 else 1.0
+        # oc-member
+        self.criminal_tendency *= 4.50 if self.oc_member and not (
+                    self.m.intervention_is_on() and self.m.oc_members_scrutinize) else 1.0
```
Comment (mariopaolucci) [inline mesa/Person.py:299]: let's get rid of `and not (self.m.intervention_is_on() and self.m.oc_members_scrutinize)`. It is useless.

**Review by mariopaolucci — CHANGES_REQUESTED** [review]
Let's integrate those small comments and pass it. I have added a commit that needs to be checked, too, so have a look.

File: mesa/Person.py [hunk 2]
```diff
@@ -266,8 +267,43 @@ def find_job(self):
             self.my_job = the_job
             the_job.my_worker = self
 
+    def factors_c(self):
```
[the remark this answers is not in the document]
Comment (Kubasinska) [reply mesa/Person.py]: I replaced with update_criminal_tendency

**Review by Kubasinska — COMMENTED** [review]
I fixed the docstrings and made the required changes.

**Review by mariopaolucci — APPROVED** [review]

**Pull request merged**